boilerplates/mantine/files/layouts/$LayoutDefault.tsx.ts

```diff
@@ -0,0 +1,40 @@
+export default async function overrideLayout() {
+  return `
+import '@mantine/core/styles.css';
+
+import React from "react";
+
+import {  MantineProvider, AppShell, Burger, Group, Skeleton, Image } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import theme from './theme';
+
+import logoUrl from "../assets/logo.svg";
+import { Link } from "../components/Link.js";
+
+export default function LayoutDefault({ children }: { children: React.ReactNode }) {
+  const [opened, { toggle }] = useDisclosure();
+  return (
+    <MantineProvider theme={theme}>
+    <AppShell
+      header={{ height: 60 }}
+      navbar={{ width: 300, breakpoint: 'sm', collapsed: { mobile: !opened } }}
+      padding="md"
+    >
+      <AppShell.Header>
+        <Group h="100%" px="md">
+          <Burger opened={opened} onClick={toggle} hiddenFrom="sm" size="sm" />
+          <a href="/"><Image h={50} fit="contain" src={logoUrl} /></a>
+        </Group>
+      </AppShell.Header>
+      <AppShell.Navbar p="md">
+        <Link href="/" label="Welcome" />
+        <Link href="/todo" label="Todo" />
+        <Link href="/star-wars" label="Data Fetching" />
+      </AppShell.Navbar>
+      <AppShell.Main>{children}</AppShell.Main>
+    </AppShell>
+    </MantineProvider>
+  );
+}
+`;
```

Comment:
Is there a way to manipulate the order which is used to layer (=replace files) the boilerplates above each other?

e.g.:
1. react
2. reac-query
3. tailwind
4. mantine

Reply:
I will add a new filename syntax supports. If a filename starts with `!`, it will replace other files on destination with the same name (removing the `!`).
The order in which files will be processed will become:
- non `!` files
- `!` files
- `$` files
- `!$` files

PR incoming.

Reply:
Done https://github.com/vikejs/bati/pull/345

Reply:
Also, there is [an `enforce` parameter](https://github.com/vikejs/bati/blob/eac9a1a940893f22fc2c67dedb9bdad3fd262195/boilerplates/shared/package.json#L30) that can take `pre` or `post` as a value for an entire boilerplate.

Reply:
Thinking more about it, using `enforce: "post"` for the `mantine` boilerplate is probably the best solution here. Be sure to rebase the repo to have the fixes of #345 though, as it ensures better predictability.

Reply:
I wanted to delete the original `layouts/style.css` which is not covered by using `!` or `$`.
What worked was using an empty file which is no where documented.

Reply:
> I wanted to delete the original `layouts/style.css` which is not covered by using `!` or `$`. What worked was using an empty file which is no where documented.

I never though of this side effect, that's why it's not documented 😅. Good enough for now, I don't have a proper solution for that yet